README.md

```diff
@@ -1,2 +1,24 @@
-# honeybee-plugin
-Use this repository as a template to write a new plugin for honeybee.
+# Ladybug Tools Template
+
+Use this repository as a template to write a new modules for Ladybug Tools.
+
+1. Use [GitHub's template repository
+   feature](https://help.github.com/en/articles/creating-a-repository-from-a-template) to
+   create a new repository from this template.
+
+2. Find and replace `{{project.name}}` with your `project name` (e.g. honeybee_therm).
+
+3. If project name includes `_` you should change the name field in `setup.py` manually.
+
+4. Update short description in `setup.py`.
+
+5. Activate the repository on [TravisCI](https://travis-ci.org/).
+  
+  ![image](https://user-images.githubusercontent.com/2915573/61428579-bbed0180-a8f0-11e9-8a4e-3ae7816c24a5.png)
+  
+  and set-up GH_TOKEN, PYPI_PASSWORD and PYPI_USERNAME.
+
+  ![image](https://user-images.githubusercontent.com/2915573/61428637-0c645f00-a8f1-11e9-9e5a-55b7a71536ed.png)
```

Comment:
6. Activate this repository on coveralls.

Particularly because it's included in the dev-requirements.txt